src/components/network/network-panel.js

```diff
@@ -129,14 +142,44 @@ genotet.NetworkPanel.prototype.setNodeInfo_ = function(node, container) {
  * @private
  */
 genotet.NetworkPanel.prototype.setEdgeInfo_ = function(edge, container) {
+  var sourceLabel, targetLabel;
+  var sourceId, targetId;
+  if (typeof edge.source == 'object') {
```

Comment:
Yes, but in renderer, for rendering purpose, when clicking an edge we can just get an edge stored in edge_ whose source and target are genotet.networkNode object. At this time we should reconstruct an genotet.networkEdge object or find that edge. So which solution is suggested?